internal/cloud/realm/dependencies.go

```diff
@@ -18,10 +19,43 @@ const (
 	dependenciesPathPattern        = appPathPattern + "/dependencies"
 	dependenciesArchivePathPattern = dependenciesPathPattern + "/archive"
 	dependenciesDiffPathPattern    = dependenciesPathPattern + "/diff"
+	dependenciesStatusPathPattern  = dependenciesPathPattern + "/status"
 
 	paramFile = "file"
 )
 
+// DependencyInstallation is used to get information from a dependencies status request
+type DependencyInstallation struct {
```

Comment:
[question] would it make sense to rename this to `DependenciesInstallation` to match the plural form of "dependencies" we're using everywhere else?  or is this "Status" really just for the installation of one dependency?  (if the latter is true, then maybe we need to rethink the name for `DependenciesStatus`)

Reply:
seems like it'd represent all dependencies given so I'll make sure plural is consistent.